app/models/MessageType.scala

```diff
@@ -35,7 +35,9 @@ object MessageType extends Enumerable.Implicits {
   case object NoReleaseForTransit         extends IeMetadata("IE051", "CC051A") with MessageType
   case object ReleaseForTransit           extends IeMetadata("IE029", "CC029A") with MessageType
 
-  val values: Seq[MessageType] = Seq(DepartureDeclaration, MrnAllocated, DeclarationRejected)
+  case object RequestOfRelease extends IeMetadata("IE054", "CC054A") with MessageType
+
+  val values: Seq[MessageType] = Seq(DepartureDeclaration, MrnAllocated, DeclarationRejected, RequestOfRelease)
```

Comment:
This is missing PositiveAcknowledgement, ControlDecisionNotification, NoReleaseForTransit and ReleaseForTransit